career/README.md

```diff
@@ -0,0 +1,52 @@
+<p align="center">
+  <img alt="logo" src="/docs/logo.png" width="200">
+</p>
+
+# 🚀 Career Documentation
+
+Have you ever wondered about what your company expects of you? How to move up in your career there?
+
+> Writer's note: well, me too. In fact, all the time! 🤔
+
+This repository is meant to make the whole process of how we handle careers at GeekHunter transparent and public.
+
+# :pushpin: Summary
+
+* [It's All About People](#its-all-about-people)
+* [Career Paths](#career-paths)
+* [Expectations](#expectations)
+
+# 😎 It's All About People
+
+At GeekHunter, we focus our efforts to create a people-based culture.
+
+Every collaborator goes through a process that we call `Ciclo de Gente`, which is a framework to develop and progress your career and skills. At its core, we rely on
```

Comment:
what do you make of `employee` instead of `collaborator`? employee conveys everyone working as a software engineer at Geek follows this process; and collaborator would mean anyone that collaborate with development in some level (such as data scientist etc)

Reply:
I think `collaborator` is a better word, especially based on your argument.

It applies to anyone at GeekHunter who also wants to join our development team, which can happen via an internal process.

Reply:
makes sense!